spec/models/bid_spec.rb

```diff
@@ -1,5 +1,31 @@
 require 'rails_helper'
 
 RSpec.describe Bid, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe "toggle accept!" do
+  	context "when bids have not been accepted" do
+	  	it "updates the bid to not be accepted" do
+	  		business_user = FactoryGirl.create(:business_user)
+	  		job = FactoryGirl.create(:job, agent_id: nil, client_id: business_user.id)
+	  		broker = FactoryGirl.create(:broker)
+	  		bid = FactoryGirl.create(:bid, job_id: job.id, bidder_id: broker.id)
+
+	  		expect(bid.accepted).to eq(false)
+	  		expect(bid.job.agent_id).to eq(nil)
```

Comment:
Awesome job with testing it :)

In this spec, since we're not triggering `toggle_accept!`, we're not really testing anything. Make sure to call the method to properly test it :)